ansible/roles/os_zabbix/vars/template_openshift_master.yml

```diff
@@ -1116,19 +1109,11 @@ g_template_openshift_master:
   # Over the course of an hour, it has run 8 times and provided 8 values for route_http_failed.
   # This trigger will alert if 2 or more of the last 5 runs have failed.
   - name: "Route has returned non-200 HTTP code multiple times in the last hour on {HOST.NAME}"
-    expression: "{Template Openshift Master:openshift.master.app.create.route_http_failed.sum(#5)}>=2"
-    url: "https://github.com/openshift/ops-sop/blob/master/v3/alerts/check_create_app.asciidoc"
-    dependencies:
-    - "Openshift Master process not running on {HOST.NAME}"
-    description: The app-create route curl check failed 2 out of the last 5 times.
-    priority: high
-
-  - name: "Service IP has returned non-200 HTTP code multiple times in the last hour on {HOST.NAME}"
-    expression: "{Template Openshift Master:openshift.master.app.create.service_http_failed.sum(#5)}>=2"
+    expression: "{Template Openshift Master:openshift.master.app.create.route_http_failed.sum(#7)}>=2"
```

Comment:
This only makes it take roughly 11 minutes longer for the alert to resolve, since it needs 2 additional successes to go back to green. It needs to be #5>=3 (or #6>=3) to actually be less sensitive.